moveit_core/robot_model/CMakeLists.txt

```diff
@@ -39,10 +39,4 @@ if(BUILD_TESTING)
   target_link_libraries(test_robot_model moveit_test_utils ${MOVEIT_LIB_NAME})
 endif()
 
-install(TARGETS ${MOVEIT_LIB_NAME}
```

Comment:
I'm not sure I really get this. It makes sense to me that it's not required for plugins but shouldn't libraries still be installed and exported?

Reply:
I missed that these are all added in the main moveit_core CMakeLists.txt, Great improvement!